React-Template-master/src/routes.tsx

```diff
@@ -1,30 +1,25 @@
 import React from 'react';
 import { Navigate, useRoutes } from 'react-router-dom';
-import { Page404 } from './pages/Page404';
 import { Login } from './pages/Login';
 import { Register } from './pages/Register';
 import { Dashboard } from './pages/Dashboard';
-import { LogoOnlyLayout } from './layouts/LogoOnlyLayout';
-import { DashboardLayout } from './layouts/dashboard';
-import { Page401 } from './pages/Page401';
-import { Page500 } from './pages/Page500';
+import { LoginAndSignUpLayout } from './layouts/LoginAndSignUpLayout';
+import { ForgotPassword } from './pages/ForgotPassword';
 
 export function Router(): React.ReactElement | null {
     return useRoutes([
         {
             path: '/dashboard',
-            element: <DashboardLayout />,
+            element: <Dashboard />,
             children: [{ path: '/dashboard', element: <Dashboard /> }]
         },
         {
             path: '/',
-            element: <LogoOnlyLayout />,
+            element: <LoginAndSignUpLayout />,
             children: [
                 { path: 'login', element: <Login /> },
                 { path: 'register', element: <Register /> },
-                { path: '404', element: <Page404 /> },
```

Comment:
Please 404 page to handle unsupported routes

Reply:
okay

Reply:
okay